howso/utilities/feature_attributes/pandas.py

```diff
@@ -556,10 +562,33 @@ def _infer_date_attributes(self, feature_name: str) -> dict:
             'date_time_format': ISO_8601_DATE_FORMAT,
         }
 
-    def _infer_time_attributes(self, feature_name: str) -> dict:
+    def _infer_time_attributes(self, feature_name: str, user_time_format: str = None) -> dict:
+        # Import this here to avoid circular import
+        from howso.client.exceptions import HowsoError
+        # First check if the user provided a format
+        if user_time_format:
+            time_format = user_time_format
+        # If not, try to infer it
+        else:
+            first_non_null = self._get_first_non_null(feature_name)
+            # If the type is datetime.time
+            if isinstance(first_non_null, datetime.time):
+                if user_time_format:
```

Comment:
I'm not sure this branch can ever be reached since it is already in an `else` branch from `if user_time_format` a layer earlier.